### PR TITLE
Configure ActionCable

### PR DESCRIPTION
**Pull request by paolomainardi**
Related to: https://github.com/sameersbn/docker-gitlab/issues/2482

### CODE REVIEW

File: assets/runtime/functions [hunk 1]
```diff
@@ -232,6 +233,17 @@ gitlab_configure_redis() {
...
 
+gitlab_configure_actioncable() {
+  echo -n "Configuring gitlab::actioncable"
+
+  gitlab_finalize_redis_parameters
+  gitlab_check_redis_connection
+
+  update_template ${GITLAB_ACTIONCABLE_CONFIG} \
+    REDIS_HOST \
+    REDIS_PORT
```
Comment (achretien) [inline assets/runtime/functions:244]: I think you need to add REDIS_DB_NUMBER here

Comment (paolomainardi) [reply assets/runtime/functions]: nice catch, added

File: assets/runtime/config/gitlabhq/cable.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# This is a template taken from here:
+# https://gitlab.com/gitlab-org/gitlab/-/blob/master/config/cable.yml.example
+development:
```
Comment (sachilles) [inline assets/runtime/config/gitlabhq/cable.yml:3]: If the definition of the Redis connection for the environments 'development' and 'test' is not needed, then I would suggest to either use other "default values" (like
`url: redis://:fakepass@fake.redis.actioncable.com:1234` and `channel_prefix: gitlab_fake_development`) or not be defined at all.

File: assets/runtime/functions (hunk 1 above)
Comment (sachilles) [reply assets/runtime/functions]: Nice work!

File: assets/runtime/config/gitlabhq/cable.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  adapter: redis
+  url: redis://localhost:6379
+  channel_prefix: gitlab_development
+test:
+  adapter: redis
+  url: redis://localhost:6379
+  channel_prefix: gitlab_test
```
Comment (sachilles) [inline assets/runtime/config/gitlabhq/cable.yml:10]: If the definition of the Redis connection for the environments 'development' and 'test' is not needed, then I would suggest to either use other "default values" (like
`url: redis://:fakepass@fake.redis.actioncable.com:1234` and `channel_prefix: gitlab_fake_development`) or these environmental setups should not be defined at all.

Comment (paolomainardi) [reply assets/runtime/config/gitlabhq/cable.yml:10]: Yes at the beginning I haven’t defined them, but then I saw this file: https://github.com/sameersbn/docker-gitlab/blob/09e76bb2e7d4c805bfc2cb70cb1b0ca89cd60216/assets/runtime/config/gitlabhq/resque.yml and replicated the same behavior. Anyway it is quite harmless to keep the other two, it’s just for consistency with other files.

Comment (sachilles) [reply assets/runtime/config/gitlabhq/cable.yml:10]: Okay, that sounds reasonable. Have you already tested this version and can you confirm that the error in the `production.log` file no longer occurs?

**Review by sachilles — APPROVED** [review]

File: assets/runtime/config/gitlabhq/cable.yml (hunk 3 above)
Comment (paolomainardi) [reply assets/runtime/config/gitlabhq/cable.yml:10]: @sachilles yes sure, using it in production now, it works like a charm.

Comment (sachilles) [reply assets/runtime/config/gitlabhq/cable.yml:10]: That sounds good. Thanks for your contribution! A patch release will follow soon.

Comment (paolomainardi) [reply assets/runtime/config/gitlabhq/cable.yml:10]: The bug affects also 14.4 and 14.5 @sachilles 

thanks for merging!

**Pull request merged**